src/components/App.js

```diff
@@ -63,48 +63,60 @@ const App = () => {
                 console.error(error);
             });
     }
-   
+
     const loadLabel = async () => {
-        const response = await github.get('labels');
+        const response = await github.get('/vue/labels');
         setLabel(response.data)
     }
 
     useEffect(() => {
         loadIssues(pageNumber);
-    }, [pageNumber, githubState]);
+    }, [pageNumber, githubState,selectedRepo]);
 
     useEffect(() => {
         loadLabel();
     },[])
 
+    function handleChange(e) {
+        setText(e.target.value);
+    }
```

Comment:
Since it's only one line, you can just use inline function
`onClick={(e) => setText(e.target.value)}`